public/angular/js/controllers/reports/index.js

```diff
@@ -39,7 +39,9 @@ angular.module('Aggie')
     $scope.statusOptions = statusOptions;
     $scope.currentPath = $rootScope.$state.current.name;
     $scope.smtcTags = smtcTags;
-    $scope.listOptions = Array.from(new Set(Object.values(ctLists.crowdtangle_list_account_pairs).flat()));
+    $scope.listOptions = Array.from(new Set(Object.values(ctLists.crowdtangle_list_account_pairs).flat())).concat(Array.from(new Set(Object.values(ctLists.crowdtangle_saved_searches).map(function(obj) {
+      return obj.name;
+    }))), ["Saved Search"]);
```

Comment:
I know you're just copying logic that was already here, worth simplifying though -- I think we can just do `Object.values({})` in both cases instead of `Array.from(new Set(Object.values({})))` since it returns the same result (or maybe I'm misunderstanding something).

Reply:
There might be duplicates in the returned array after `flat` operation. So using `new Set` would only keep the unique values which is what we want. I think this is fine unless theres a better way to find unique values and make an array.

Reply:
Ah okay, yes there are easier ways with Lodash for example but not worth blocking this. Thanks for clarifying what's going on.

Reply:
Yeah, I wrote the original line there. I think I just didn't know we had lodash at the time.

We might want to add a note to refactor this, because it's definitely verbose, if functional.